nixos/lib/systemd-lib.nix

```diff
@@ -524,7 +524,9 @@ rec {
           # Stupid misc. symlinks.
           ln -s ${cfg.defaultUnit} $out/default.target
           ln -s ${cfg.ctrlAltDelUnit} $out/ctrl-alt-del.target
```

Comment:
Possibly remove these as well / replace with alternatives? see https://github.com/NixOS/nixpkgs/pull/419015#pullrequestreview-3111792107

Reply:
For clarity, I think we should Nixify these defaults but I think it’s not really in scope for this PR – I just think that if we’re not setting up `kbrequest.target` by default we shouldn’t set it at all here, in favour of our normal systemd configuration mechanisms.